src/status_im/utils/number.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.utils.number
+  "Contains helpers for numbers"
+  (:require [clojure.string :refer [upper-case]]))
+
+(defn- number-to-nk [number-string capitalize?]
+  (let [numeric-value (if (> 0 number-string)
+                        (throw (str "Invalid input " number-string))
+                        (js/parseInt number-string))]
+    (if-not (js/isNaN numeric-value)
+      (if (<= 1000 numeric-value)
```

Comment:
kinda hard to read, when number is the first argument, why not `(> numeric-value 1000)` ?

Reply:
Fixed, Thanks Andrey